### PR TITLE
expose an API to get the DeckLink SDK version number

### PR DESCRIPTION
**Pull request by lzuosym**

### CODE REVIEW

**Comment by lzuosym** [comment]
@ccbrown, I can click the merge button without approval, please add some protection.

**Review by ccbrown — COMMENTED** [review]
This looks good to me, but how will users of this library create an `APIInformation`? It seems like you'll need to add something to do that, right?

**Comment by lzuosym** [comment]
> APIInformation

You're right. I missed it!

File: src/lib.rs
Comment (ccbrown) [inline src/lib.rs:1636]: I see you added a binding, but there's still no public way to create one of these right?

```suggestion
impl APIInformation {
    pub fn new() -> Self {
        unsafe {
            Self{
                implementation: create_decklink_api_information_instance(),
            }
        }
    }
```

Comment (lzuosym) [reply src/lib.rs:1636]: Absolutely right! I am sure I will get this problem when using `APIInformation` in `avs-agent`.  Should it return a `Result` type in case `create_decklink_api_information_instance` fails?

Comment (ccbrown) [reply src/lib.rs:1636]: Yeah, probably. Or maybe an Option would make more sense since we wouldn't have an error code if it fails.

Comment (lzuosym) [reply src/lib.rs:1636]: okay

**Review by ccbrown — APPROVED** [review]

**Pull request merged**
